### PR TITLE
Onward journey

### PR DESCRIPTION
**Pull request by emmalewis**
- Adds onward journey
- Creates an environment variable which defines the relatedContent list to display (e.g. 'thing/adae44ca-4ea7-3cf8-8332-bf85ec89a558') and renders some onward journey content if that is present.

Screenshot 2022-04-28 at 18 20 33
Screenshot 2022-04-28 at 18 21 14

### CODE REVIEW

File: views/layout.html [hunk 1]
```diff
@@ -53,7 +53,7 @@
         Serve a different stylesheet for IE8 :
         http://www.theguardian.com/info/developer-blog/2013/oct/14/mobile-first-responsive-ie8 #}
         <!--[if gt IE 8]><!-->
-        <link rel="stylesheet" href="{{ asset('styles/main.css') }}?v=22">
+        <link rel="stylesheet" href="{{ asset('styles/main.css') }}?v=23">
```
Comment (JALWilliams) [inline views/layout.html:56]: I think we need to delete the query string at the end (ex/ ?v=23)

File: views/layout.html [hunk 2]
```diff
@@ -263,7 +263,7 @@ <h3 class="o-footer__matrix-title" aria-controls="o-footer-section-4">
         </footer>
 
         <script src="https://origami-build.ft.com/v3/bundles/js?modules=o-hoverable@^3.2,o-fonts@^5.3,o-forms@^9.3,o-colors@^6.4,o-grid@^6.1,o-buttons@^7.5,o-footer@^9.2"></script>
-        <script src="{{ asset('scripts/main.bundle.js') }}?v=1"></script>
+        <script src="{{ asset('scripts/main.bundle.js') }}?v=2"></script>
```
Comment (JALWilliams) [inline views/layout.html:266]: Do we want to delete the query string at the end here too?

File: views/layout.html (hunk 1 above)
Comment (emmalewis) [reply views/layout.html:56]: @JALWilliams We ended up needing the query string on the live site to stop the resources from caching too. Not sure how it ever worked previously, but I definitely needed it once I deployed to live, so just need to remember to increment value whenever we change js or css.

**Review by JALWilliams — APPROVED** [review]
LGTM!

**Pull request merged**